ocs_ci/cleanup/aws/cleanup.py

```diff
@@ -131,28 +131,47 @@ def aws_cleanup():
         required=False,
         help="The name of the AWS region to delete the resources from"
     )
+    parser.add_argument(
+        '--force',
+        action='store_true',
+        required=False,
+        help="Skip confirmation prompt. WARNING: this utility is destructive, "
```

Comment:
@clacroix12 ,
For me its not clear about what confirmation prompt will be skipped! Can this be more explicit ?

Reply:
Done